### PR TITLE
Build the homepage

### LINKED ISSUE #2: Todo: Build the Homepage

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
This Pull Request contains

- ProductCard component
- Styling all the pages 
- Added products and categories in the backend
Screenshot (110)
Screenshot (111)

### CODE REVIEW

File: src/components/ProductCard.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <div className="image">
+          <img src={imageUrl} alt={title} />
+        </div>
+        <div className="card-heading-container">
+          <h3 className="font-md font-medium">{storeName}</h3>
+          <h5 className="font-sm font-light p-b-1">{title}</h5>
+        </div>
+        <div className="card-badge card-badge-position">{category}</div>
+        <div className="favourite-icon favourite-icon-position">
+          <a className="btn btn-link-noBorder" href>
```
Comment (sadathshariff) [inline src/components/ProductCard.jsx:14]: use link tag here

File: src/styles/Home.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
+
+/* .banner-image{
+    background-image: url(/assets/gourmet\ bg.jpg);
+    height: calc(100vh - 5rem);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: relative;
+
+} */
```
Comment (sadathshariff) [inline src/styles/Home.css:10]: remove the commented code

File: src/styles/main.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
+
+/* @import url("/gourmet_react/src/styles/authentication.css");
+@import url("/gourmet_react/src/styles/cart.css");
+@import url("/gourmet_react/src/styles/Home.css");
+@import url("/gourmet_react/src/styles/product-listing.css");
+@import url("/gourmet_react/src/styles/variable.css");
+@import url("/gourmet_react/src/styles/wishlist.css"); */
```
Comment (sadathshariff) [inline src/styles/main.css:7]: here as well

**Review by sadathshariff — APPROVED** [review]
The Code looks good

File: src/styles/Home.css (hunk 2 above)
Comment (vaibhav-mahalle) [reply src/styles/Home.css:10]: thanks for pointing it out removed it.

File: src/styles/main.css (hunk 3 above)
Comment (vaibhav-mahalle) [reply src/styles/main.css:7]: removed the commented code.

**Review by Gautam2305 — APPROVED** [review]
everything looks good, go for merge.
I liked how you specified everything in the backend for your products.

File: src/components/ProductCard.jsx (hunk 1 above)
Comment (vaibhav-mahalle) [reply src/components/ProductCard.jsx:14]: will look into it thanks for the suggestion.

**Pull request merged**